package.json

```
@@ -36,13 +43,31 @@
     "glob-all": "^3.2.1",
     "html-loader": "^1.1.0",
     "html-webpack-plugin": "^4.3.0",
+    "ignore-loader": "^0.1.2",
+    "imports-loader": "^1.1.0",
+    "karma": "4.2.0",
+    "karma-browserify": "^7.0.0",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-coverage": "1.1.2",
+    "karma-firefox-launcher": "1.1.0",
+    "karma-mocha": "1.1.1",
+    "karma-rawfixtures-preprocessor": "linagora/karma-rawfixtures-preprocessor",
+    "karma-sinon-chai": "^2.0.2",
+    "karma-spec-reporter": "0.0.26",
+    "karma-webpack": "^4.0.2",
     "less-loader": "^6.1.1",
     "mkdirp": "1.0.4",
+    "mocha": "6.2.0",
+    "mockery": "1.7.0",
     "pug": "^3.0.0",
     "pug-html-loader": "^1.1.5",
     "pug-loader": "^2.4.0",
+    "q": "^1.5.1",
     "replace-in-file": "^6.1.0",
+    "restangular": "^1.6.1",
     "rimraf": "^3.0.2",
+    "sinon": "1.17.5",
+    "sinon-chai": "2.8.0",
```

```suggestion
    "sinon": "6.1.4",
    "sinon-chai": "3.2.0",
```

You should use the packages' versions that were previously used in the old codebase. Using such old versions will likely lead to trouble in the future.

---

@huy-ta I reckon that Ngoc also uses this version in his PR for setup test. I try to change to your suggestion and apparently some test case will fail because of the change

---

My bad, for inbox, [we were using the newer version of `sinon`](https://github.com/linagora/linagora.esn.unifiedinbox/blob/764de58c53d05fd0bf96f0cd8ca1e567e8bd19dd/package.json#L45), but for admin, [we were using the old version of `sinon`](https://github.com/linagora/linagora.esn.admin/blob/e69a91a7fccdf7b66b4d512234b0a36deae06e3f/package.json#L61).